spacy/pipeline/entity_linker.py

```diff
@@ -174,9 +183,14 @@ def __init__(
             Scorer.score_links.
         use_gold_ents (bool): Whether to copy entities from gold docs or not. If false, another
             component must provide entity annotations.
-
+        abstention_threshold (Optional[float]): Confidence threshold for entity predictions. If confidence is below the
+            threshold, prediction is discarded. If None, predictions scores are not checked.
         DOCS: https://spacy.io/api/entitylinker#init
         """
+
+        if abstention_threshold is not None and not (0 <= abstention_threshold <= 1):
```

Comment:
Should this be strictly less than 1?

Reply:
Don't have a strong opinion on this. Do you think it should be? I think we could leave that up to the user.

Reply:
No, I don't think it makes sense to strictly enforce that. Using 1 would be kind of silly but the user has to choose it.